assets/maps/prefabs/space/prefab_hunting.dmm

```diff
@@ -0,0 +1,502 @@
+"s" = (
+/obj/machinery/light/incandescent/blueish{
+	dir = 8
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"t" = (
+/obj/decal/stripe_delivery,
+/obj/table/reinforced/industrial/auto,
+/obj/item/pipebomb/frame{
+	pixel_y = 5
+	},
+/obj/item/cable_coil/yellow,
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"u" = (
+/obj/machinery/light/small/floor/warm,
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"v" = (
+/obj/machinery/door/airlock/pyro/glass/mining{
+	requires_power = 0
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"w" = (
+/obj/rack,
+/obj/item/clothing/head/NTberet,
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"x" = (
+/obj/decoration/vent/vent3{
+	pixel_y = 28
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"y" = (
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"z" = (
+/obj/machinery/light/incandescent/blueish{
+	dir = 4
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"A" = (
+/obj/random_pod_spawner/random_putt_spawner,
+/turf/simulated/floor/engine,
+/area/prefab/huntingsupplies)
+"C" = (
+/turf/simulated/floor/engine,
+/area/prefab/huntingsupplies)
+"D" = (
+/obj/machinery/light/small/blueish,
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"E" = (
+/obj/decal/poster/flag/progressive{
+	pixel_y = 32
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"F" = (
+/obj/item/storage/secure/ssafe{
+	pixel_y = 32;
+	spawn_contents = list(/obj/item/gun/kinetic/sawnoff/birdshot);
+	name = "gun safe"
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"H" = (
+/obj/machinery/door_control{
+	id = "hunting_supplies";
+	pixel_y = 28
+	},
+/obj/machinery/light/small/floor/netural,
+/turf/simulated/floor/engine,
+/area/prefab/huntingsupplies)
+"I" = (
+/obj/decal/poster/wallsign/no_smoking{
+	pixel_x = -32
+	},
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"J" = (
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"K" = (
+/obj/machinery/light/small/floor/netural,
+/turf/simulated/floor/engine,
+/area/prefab/huntingsupplies)
+"L" = (
+/obj/rack,
+/obj/item/storage/pouch/highcap/midcap,
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"M" = (
+/obj/machinery/space_heater,
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"P" = (
+/turf/simulated/floor/engine/caution/east,
+/area/prefab/huntingsupplies)
+"Q" = (
+/obj/machinery/door/poddoor/blast/pyro/podbay_autoclose{
+	dir = 4;
+	id = "hunting_supplies";
+	name = "Big Bill Hell's Hunting Supplies"
+	},
+/turf/simulated/floor/engine,
+/area/prefab/huntingsupplies)
+"R" = (
+/obj/decal/stripe_delivery,
+/obj/table/reinforced/industrial/auto,
+/obj/item/storage/toolbox/mechanical/orange_tools,
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"S" = (
+/obj/reagent_dispensers/watertank/fountain{
```

Comment:
Watercooler is visually aligned so that it seems like you can walk past it, but it's impassable, meaning you have to shuffle awkwardly around.